server/server.ts

```diff
@@ -103,4 +103,29 @@ app.get("/api/generateDaily",(req, res) => {
     new dailyEntriesModel(daily).save();
   }
   res.send({message:"generated 100 datas!"});
-})
\ No newline at end of file
+})
+
+//Below is a post request for the users to register
+app.post("/api/register", (req, res) => {
+  //console.log('req here:!!', req.body);
+  let userData = req.body;
+  let userReg = new userEntriesModel({
+    firstName: userData.firstName,
+    lastName: userData.lastName,
+    age: +userData.age,
+    gender: userData.gender.value,
+    email: userData.email,
+    phoneNumber: +userData.phoneNumber,
+    height: +userData.height,
+    weight: +userData.weight,
+    caloriesGoal: +userData.targetCalories,
```

Comment:
would we want some to use a suggested calorie goal if this is not provided or is it on the front end?

Reply:
Good call! On server.ts and the userSchema, I added caloriesRecommanded. I will be doing the math on the front end to figure out the recommended calories based on the BMI, and then I will send the number to the database to store.